src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/client/HmppsAuthApiClient.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.welcometoprison.client
```

Comment:
noticed that the BASM client and service are under /model/basm instead of being under top-level client/service packages. Happy to move these classes into the same structure if that's what we're going with 😄